src/Dialogs/Weather/SkysightDialog.cpp

```diff
@@ -0,0 +1,376 @@
+/*
```

Comment:
Copyright licenses should be replaced with SPD placeholders like all other files.